client-ts/FunctionalTests/ts/TestLogger.ts

```diff
@@ -0,0 +1,17 @@
+import { ILogger, LogLevel } from "@aspnet/signalr";
+
+export class TestLogger implements ILogger {
+    public static instance: TestLogger = new TestLogger();
```

Comment:
Globals ftw ;). It's the easiest way to deal with this in JavaScript and it's safe because Jasmine doesn't run tests in parallel because JavaScript is single-threaded.

Reply:
Had an idea for a simple condition to verify this remains the case (that Jasmine only runs a single test at a time), since that will mess with `TestLogger`